core/src/main/java/fr/sncf/osrd/conflicts/SpacingResourceGenerator.kt

```diff
@@ -239,17 +216,16 @@ class SpacingRequirementAutomaton(
                         0,
```

Comment:
I believe a lot of the setup you're doing could be avoided by using the start block parameter of simulator.evaluate. If it turns out to not be useful, we could also remove the parameter entirely and make the code simpler